### PR TITLE
Updated Dockerfile to build openvas v20.8.0

### PR DESCRIPTION
**Pull request by emerlinsky**

### CODE REVIEW

**Comment by TheDoctor0** [comment]
I see that there were multiple changes made with 20.8 releases. I need to test this to be sure that everything still work as intended.

**Comment by emerlinsky** [comment]
I am running tests and working on catching issues now. I will push the results in my branch when I am done.

**Comment by TheDoctor0** [comment]
To be honest, I wasn't expecting that. Great!
If you confirm that there are no issues, I'll merge this PR.

**Comment by emerlinsky** [comment]
That was not a piece of cake thing that I expected it to be originally :) But it works fine in this iteration.

File: README.md [hunk 1]
```diff
@@ -35,19 +40,11 @@ By default report is saved as *openvas.report*.
 #### Formats
 
 1. Anonymous XML
-2. ARF
```
Comment (TheDoctor0) [inline README.md:38]: I see that multiple report formats were removed.
That may cause issues for people that use this container in an automatic scan and report pipeline.
I guess I'll have to make a legacy branch for OpenVAS 10, same as for OpenVAS 9 with HTML format.

Comment (emerlinsky) [reply README.md:38]: Yes, also some of the predefined port lists were removed too. Besides, port list id or port/ports need to be explicitly passed to the scanner now.

Comment (iacopo-papalini) [reply README.md:38]: By Greenbone documentatioon page, ARF is supported by OpenVAS 20.08: https://docs.greenbone.net/GSM-Manual/gos-20.08/en/reports.html#configuring-and-managing-report-formats

Is there a reason I am missing you removed it or is it just an oversight?

File: README.md [hunk 2]
```diff
@@ -26,6 +26,11 @@ This will start up the container and update the NVTs cache - it can take some ti
 
 After that, the scan script will run and the progress will be displayed in the console.
 
+
+#### Target
+
+Target can be a single IP or a comma separated list of IP addresses.
```
Comment (iacopo-papalini) [inline README.md]: In previous versions, you could send a 'netmasked' IP like 192.168.1.0/24 and it worked. Did the behavior change?

Comment (emerlinsky) [reply README.md]: You still should be able to specify a subnet range using CIDR notation. I will update the README to reduce confusion.

File: README.md (hunk 1 above)
Comment (emerlinsky) [reply README.md:38]: It seems that their documentation might be out of date. More on that can be read here:
https://community.greenbone.net/t/gvm-20-08-missing-report-formats-and-scan-configs/6397/6

**Comment by TheDoctor0** [comment]
I'll try to take some time this weekend to build and test it - just to be sure as changes are quite substantial.

**Comment by TheDoctor0** [comment]
@emerlinsky, I made some changes on the current branch to fix the building with TravisCI and make it much faster.
If you want you can resolve merge conflicts by yourself, but I will do it in the coming days anyway.

**Pull request merged**